readme.md

```diff
@@ -900,6 +900,7 @@
 - [Computational Geometry](https://github.com/atkirtland/awesome-computational-geometry#readme) - Computational approaches for problems in geometry.
 - [OpenStreetMap](https://github.com/osmlab/awesome-openstreetmap#readme) - An open data mapping project utilized by many apps and devices.
 - [Computational Biology](https://github.com/inoue0426/awesome-computational-biology#readme) - Computational approaches applied to problems in biology.
+- [Quarto](https://github.com/mcanouil/awesome-quarto#readme) - Quarto® is an open-source scientific and technical publishing system built on Pandoc, allowing computations using Jupyter or R/knitr.
```

Comment:
The description should not include the list name.

Reply:
Apologies, this has been fixed in 0b4af81138d453c22eea3d661f222f13816179e3